nbextensions/appCell2/widgets/tabs/jobStateViewer.js

```diff
@@ -26,14 +26,26 @@ define([
                 label = 'success';
                 color = 'green';
                 break;
+            case 'finished':
+                label = 'success';
+                color = 'green';
+                break;
             case 'suspend':
                 label = 'error';
                 color = 'red';
                 break;
+            case 'error':
+                label = 'error';
+                color = 'red';
+                break;
             case 'canceled':
```

Comment:
Is "canceled" still an option from the API? What are the states that are available?

Reply:
no. `canceled` status is no longer available. I updated the state viewer code to remove all `canceled` instances.